python/lsst/ts/imsim/utils/utility.py

```diff
@@ -95,9 +95,11 @@ def get_camera(inst_name: str) -> cameraGeom.Camera:
     # Check the input
     if (inst_name == "lsstfam") or (inst_name == "lsst"):
         return LsstCam().getCamera()
+    elif inst_name == "comcam":
+        return LsstComCam().getCamera()
     else:
         raise ValueError(
-            f"This instrument name ({inst_name}) is not supported. Must be 'lsstfam' or 'lsst'."
+            f"This instrument name ({inst_name}) is not supported. Must be 'lsstfam', 'lsst', or 'comcam'."
```

Comment:
Is the test for `get_camera` function also updated?

Reply:
Yes, it is in this PR.